src/coreclr/src/System.Private.CoreLib/src/System/Threading/Thread.CoreCLR.cs

```diff
@@ -7,6 +7,7 @@
 using System.Runtime.InteropServices;
 using System.Runtime.Serialization;
 using System.Runtime.Versioning;
+using Internal.Runtime.CompilerServices;
```

Comment:
Why do we need this? `IntrinsicAttribute` is under `System.Runtime.CompilerServices`

Reply:
Ah, right, removed.